### ISSUE TITLE
Question(s)

### ISSUE THREAD

**Issue by Fireworkstars46** [Pokechu22/WorldDownloader#269]
Is there a forge mod for this on 1.16.1?

**Comment by Pokechu22** [comment]
No. A forge version was included with 1.16.2 in [v4.1.0.2](https://github.com/Pokechu22/WorldDownloader/releases/tag/v4.1.0.2), and dropped in [v4.1.1.0](https://github.com/Pokechu22/WorldDownloader/releases/tag/v4.1.1.0); 1.16.3 and 1.16.4 are still supported. I don't remember exactly why I never supported forge 1.16.1.

**Comment by Fireworkstars46** [comment]
Can you add the 1.16.1 forge one? Cause I only play 1.16.1 and older (any version from 1.16.2 and above I won't play)
